lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1529,6 +1529,40 @@ OpFoldResult Aten__Contains__StrOp::fold(ArrayRef<Attribute> operands) {
   return nullptr;
 }
 
+//===----------------------------------------------------------------------===//
+// Aten__Contains__IntListOp
+//===----------------------------------------------------------------------===//
+
+static bool isListConstructNotModified(Value torchList) {
+  if (!llvm::all_of(torchList.getUsers(), [](Operation *op) {
+        return isa<Aten__Contains__IntListOp>(op);
+      }))
+    return false;
+
+  return torchList.getDefiningOp<Torch::PrimListConstructOp>();
```

Comment:
This can be written as:

```cpp
return llvm::all_of(torchList.getUsers(), ...);
```

Reply:
I don't understand this, all_of is an iterator, not a returnable function, right? How could I return this function as a boolean?

Reply:
Nevermind, it's an iterator that returns a function, my mistake.